### DISCUSSION TITLE
For DropDown component，can label and field be arranged horizontally

### DISCUSSION THREAD

**Discussion by Hello-dl** [carbon-design-system/carbon#11660] [Help] [answered]
I'm using DropDown component, but seems that label can only be above field，would like to know, is there a way to put label on the left of field?

**Comment by tay1orjones** [comment, marked as answer]
Hi @Hello-dl, yes we call this configuration "[inline](https://react.carbondesignsystem.com/?path=/story/components-dropdown--inline)" and can be configured using the prop `type="inline"`

**Reply by Hello-dl** [reply]
It worked! Thank you so much!
